py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -224,13 +227,16 @@ def populate_los_ids(self, forests):
                 w = forest.ivar>0
                 weights = np.empty_like(forest.log_lambda)
                 weights[w] = 1
-                weights[~w]= 0
-                mean_expected_flux = forest.continuum
+                weights[~w] = 0
             else:
-                mean_expected_flux = forest.continuum
                 weights = 1. / self.compute_forest_variance(
                     forest, forest.continuum)
 
+            mean_expected_flux = forest.continuum
```

Comment:
We may want to use `mean_expected_flux = np.copy(forest.continuum)` to have same behavior as fitting. Explicit copying keeps continuum unmodified, which is the default behavior for fitting.

Reply:
Makes sense to me